generator/transit_generator.hpp

```diff
@@ -102,15 +104,74 @@ class DeserializerFromJson
   OsmIdToFeatureIdsMap const & m_osmIdToFeatureIds;
 };
 
-/// \brief Builds the transit section in the mwm.
+/// \brief The class contains all the information to make TRANSIT_FILE_TAG section.
+class GraphData
+{
+public:
+  void DeserializeFromJson(my::Json const & root, OsmIdToFeatureIdsMap const & mapping);
+  void SerializeToMwm(std::string const & mwmPath) const;
+  void AppendTo(GraphData const & rhs);
+  void Clear();
+  bool IsValid() const;
+
+  /// \brief Sorts all class fields execpt for |m_edges| by their ids.
+  void Sort();
+  /// \brief Calculates and updates |m_edges| by adding valid value for Edge::m_weight
+  /// if it's not valid.
+  /// \note |m_stops|, Edge::m_stop1Id and Edge::m_stop2Id in |m_edges| should be valid before call.
```

Comment:
s/should/must/ ?
Are you guessing or is it a requirement?

Reply:
must. It's a requirement.